docs/trino-connector/installation.md

```diff
@@ -74,11 +74,13 @@ To configure Gravitino connector correctly, you need to put the following config
 connector.name=gravitino
 gravitino.uri=http://gravitino-server-host:8090
 gravitino.metalake=test
+gravitino.simplify-catalog-names=true
 ```
 
 - The `gravitino.name` defines which Gravitino connector is used. It must be `gravitino`.
 - The `gravitino.metalake` defines which metalake are used. It should exist in the Gravitino server.
 - The `gravitino.uri` defines the connection information about Gravitino server. Make sure your container can access the Gravitino server.
+- The `gravitino.simplify-catalog-names` setting omits the metalake prefix from catalog names when set to true. If you set it to true, Trino will configure only one Graviton catalog.
```

Comment:
What's meaning of `will configure only one Graviton catalog.`, do you mean that user cannot configure other catalogs out of gravitino?

Reply:
Besides, can you change this use table format.